### ISSUE TITLE
Does the collector support collecting metrics from Google Cloud Platform ? 

### ISSUE THREAD

**Issue by chenlujjj** [open-telemetry/opentelemetry-collector-contrib#26559]
### Describe the issue you're reporting

Google Cloud Monitoring provides lots of metrics: https://cloud.google.com/monitoring/api/metrics, and we want to use otel collector to collect them and write to the backend storage we are using. But it seems that now otel collector doesn't support collecting GCP metrics yet. I've searched around in this repo but haven't found such a receiver. Please correct me if I missed something here.

The [grafana agent](https://grafana.com/docs/agent/latest/static/configuration/integrations/gcp-exporter-config/) has a similar feature. Maybe that can be a reference if we want the otel collector to have this capability.

**Comment by codeboten** [comment]
@dashpole @aabmass is there a receiver that can be used for this today?

**Comment by dashpole** [comment]
Duplicate of https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2286.

TL;DR no receiver exists today, but we (Google) are happy to help guide anyone who is able to contribute one. I have volunteered as a sponsor for it, if someone steps up to implement and support it.

**Comment by adriangb** [comment]
@dash do you know if such a thing has ever materialized? Could you offer some basic guidance on how you’d shape the solution so help determine if it’s something I/we could tackle or not?

**Comment by dashpole** [comment]
Yes it did.  See https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/googlecloudmonitoringreceiver
